packages/nimble-components/src/rich-text-viewer/testing/rich-text-viewer.pageobject.ts

```diff
@@ -0,0 +1,15 @@
+import type { RichTextViewer } from '..';
+
+/**
+ * Page object for the `nimble-rich-text-viewer` component.
+ */
+export class RichTextViewerPageObject {
+    public constructor(
+        private readonly richTextViewerElement: RichTextViewer
+    ) {}
+
+    public getFirstChildElement(): Element | null | undefined {
```

Comment:
The goal here is to get the rendered children right? Can we call this something like `getFirstMarkdownRenderedElement`

Reply:
Yes, I have updated the function name as suggested.